src/rohan/VowelsFinder.java

```diff
@@ -0,0 +1,53 @@
+package rohan;
+
+import java.util.Scanner;
+
+public class VowelsFinder {
+	int aCnt,eCnt,iCnt,oCnt,uCnt;
+	
+	void findVowels(String str) {
+		int cnt=0;
+		String vowels = "";
+		for(int i=0;i<str.length();i++) {
+			char ch = str.charAt(i);
+			switch(ch) {
+				case 'A':
+				case 'a': 
+						if(++aCnt==1)
+							vowels += ch+",";
```

Comment:
what is use of aCnt++==1 here?

Reply:
Hi Harsh, 
I've used aCnt, eCnt, iCnt etc to maintain the count of vowels in the string and print the total count at the end. However I can also print individual vowel count.